rubicon-htb.js

```diff
@@ -254,6 +256,40 @@ function RubiconModule(configs) {
         return firstPartyData;
     }
 
+    function _getDigiTrustQueryParams() {
+        if (configs.digitrustId) {
+            _digiTrustId = configs.digitrustId;
```

Comment:
does `_digiTrustId` need to be set?  it's only set if `configs.digitrustId` is set, and thinking maybe you can just use `configs.digitrustId` directly, unless this could be called again after the `_digiTrustId` global has already been set when `configs.digitrustId` is no longer set.   if that's not the case, could probably just get rid of `_digiTrustId` and later do `var digiTrustId = configs.digitrustId || getDigiTrustId();` (see later comments as well)

Reply:
Yes I did it for visibility but agree it is better. However I would advise to sill use the getDigiTrustId() so we could avoid calling all the JS (_window + try catch) in case configs.digitrustId is defined